aws-rds-dbsubnetgroup/aws-rds-dbsubnetgroup.json

```diff
@@ -12,7 +12,7 @@
     },
     "SubnetIds": {
       "type": "array",
-      "uniqueItems": true,
+      "uniqueItems": false,
```

Comment:
Why do we need to update this if we change this? The customers input SubnetIDs are already converted into a set which should enforce uniqueness.

Reply:
Because of backward compatibility. Current stacks have duplicate `SubnetIds` and working fine. If we did not change it, We would break already working stacks.